R/sof.mpm2.R

```diff
@@ -12,6 +12,12 @@
 #'   of elliptically shaped peaks.
 #' @param peak.shape [\code{character(1)}]\cr
 #'   Shape of peak(s). Possible values are \dQuote{ellipse} and \dQuote{sphere}.
+#' @param evaluation.env [\code{character(1)}]\cr
+#'   Evaluation environment after the function was created. Possible values are
+#'   \dQuote{R} (default) and \dQuote{Python}. The original generation of the
```

Comment:
Hmm, the case-sensitivity here is actually not user-friendly. Should we allow arbitrary string? E.g., r or R or pYthon or PyThOn?

Reply:
Good catch, I made it case-insensitive now